[Gst-va] add transcode hevc and mpeg2 support

FocusLuo opened the pull request:
add transcode support for gst-va hevc and mpeg2 input support
1, hevc 8bit input
2, mpeg2 input

uartie reviewed (changes requested):
Please provide a PR for BAT test HEVC/MPEG2 xcode references to avoid breaking CI pre-check for others.  I have already fixed the AVC xcode case in BAT that got merged previously.

FocusLuo commented:
> Please provide a PR for BAT test HEVC/MPEG2 xcode references to avoid breaking CI pre-check for others. I have already fixed the AVC xcode case in BAT that got merged previously.

Added, (Comments: gst-va doesn't support mpeg2 encode, so this will not impact gst-va bat mpeg2 transcode)
